internal/services/datadog/azurerm_datadog_monitor_sso_configuration.go

```diff
@@ -80,6 +79,23 @@ func resourceDatadogSingleSignOnConfigurations() *pluginsdk.Resource {
 			},
 		},
 	}
+
+	if !features.FivePointOh() {
+		resource.Schema["single_sign_on"].Required = false
+		resource.Schema["single_sign_on"].Optional = true
```

Comment:
We'll want these to be computed as well since we're setting both into state even though the user will only specify one

Reply:
Also, at this point, we could just have rewrite this with another block instead of changing the existing block

```
resource.Schema["single_sign_on"] = &pluginsdk.Schema{
...
}
```